client/main.go

```diff
@@ -68,6 +68,7 @@ func Setup() {
 	HardwareClient = hardware.NewHardwareServiceClient(conn)
 }
 
+// NewTinkerbellClient : Function to get a new hardware client
```

Comment:
Please change the format to be more conversational, for example:
```
NewTinkerbellClient creates a new hardware client
```

Reply:
Done.